package.json

```
@@ -2,7 +2,7 @@
   "name": "@morpho-org/morpho-blue-bundlers",
   "description": "Morpho Blue Bundlers",
   "license": "GPL-2.0-or-later",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

```suggestion
  "version": "1.1.1",
```

It's only a deps bump!